mip/mip.c

```diff
@@ -313,8 +313,9 @@ static void arp_ask(struct mip_if *ifp, uint32_t ip) {
 
 static void onstatechange(struct mip_if *ifp) {
   if (ifp->state == MIP_STATE_READY) {
+    struct mg_addr addr = {};
```

Comment:
Looks like some compilers don't like this construction as it seems to be a GNU extension

Reply:
Yeah, fixing those ...